src/presenter/trip-presenter.js

```diff
@@ -22,40 +22,43 @@ export default class TripPresenter {
   }
 
   #renderPoint(point, allOffers, destinations) {
-    const pointComponent = new PointView({ point, allOffers, destinations});
-    const pointEditComponent = new EditPointView({ point, allOffers, destinations});
 
-    const replaceCardToForm = () => {
-      this.#pointListContainer.element.replaceChild(pointEditComponent.element, pointComponent.element);
-    };
+    const pointComponent = new PointView({ point,
+      allOffers,
```

Comment:
Здесь в данном случае лучше и 1 параметр переносить
```suggestion
    const pointComponent = new PointView({
      point,
      allOffers,
```